LoopKitUI/Models/ColorPalette.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ColorPalette.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-07-29.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public struct ColorPalette {
+    public var theColor: Color?
```

Comment:
(and similarly I think the colors within the palette should be non-optional as well)